assets/js/cookie.js

```diff
@@ -22,7 +22,8 @@ function getCookie(name) {
 }
 
 function deleteCookie(name) {
-	document.cookie = name + '=; Max-Age=-99999999;';
+	document.cookie = name + '=;Max-Age=-99999999;path=/;';
+	document.cookie = name + '=;Max-Age=-99999999;path=/;domain=.tomatenkuchen.eu;';
```

Comment:
This probably isn't intended?

Reply:
Has to be this way because cookies across subdomains are weird. Dont ask, thx.